cmd/geth/snapshot.go

```diff
@@ -220,6 +221,26 @@ block is used.
 				Description: `
 The export-preimages command exports hash preimages to a flat file, in exactly
 the expected order for the overlay tree migration.
+`,
+			},
+			{
+				Name:      "traverse-var",
```

Comment:
How about `inspect-storage` something like that?

Reply:
fixed